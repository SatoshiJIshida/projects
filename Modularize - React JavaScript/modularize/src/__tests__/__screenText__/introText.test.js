import React from "react";
import { shallow } from "enzyme";
import IntroText from "../../components/screenText/introText";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Intro Text", () => {
  it("should be defined", () => {
    expect(IntroText).toBeDefined();
  });

  it("should render correctly", () => {
    const introText = shallow(<IntroText name="introText-test" />);
    expect(introText).toMatchSnapshot();
  });
});
