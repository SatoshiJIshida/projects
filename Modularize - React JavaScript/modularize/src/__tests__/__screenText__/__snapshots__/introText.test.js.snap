// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro Text should render correctly 1`] = `ShallowWrapper {}`;
